testament/lib/stdtest/testutils.nim

```diff
@@ -26,11 +26,22 @@ template flakyAssert*(cond: untyped, msg = "", notifySuccess = true) =
     echo msg2
 
 proc greedyOrderedSubsetLines*(lhs, rhs: string): bool =
-  ## returns true if each stripped line in `lhs` appears in rhs, using a greedy matching.
-  let rhs = rhs.strip
-  var currentPos = 0
+  ## Returns true if each stripped line in `lhs` appears in rhs, using a greedy matching.
```

Comment:
add:
```nim
  doAssert not greedyOrderedSubsetLines("a1\na5", "a0\na1\na2\na3\na4\nprefix:a5")
  doAssert not greedyOrderedSubsetLines("a1\na5", "a0\na1\na2\na3\na4\na5:suffix")
```
to the existing `tests/stdlib/ttestutils.nim`

Reply:
fixed